configuration/ibm/50001001.json

```diff
@@ -929,6 +929,12 @@
             {
                 "inventoryPath": "/xyz/openbmc_project/inventory/system/chassis/motherboard/lcd_op_panel_hill",
                 "serviceName": "xyz.openbmc_project.Inventory.Manager",
+                "pollingRequired": {
+                    "gpioPresence": {
+                        "pin": "RUSSEL_OPPANEL_PRESENCE_N",
+                        "value": 0
```

Comment:
what does value signify ?

Reply:
The value indicates the expected state of the presence pin

Reply:
This will be updated with story [PFEBMC-3295](https://jsw.ibm.com/browse/PFEBMC-3295)